churaverse-plugins-server/src/churarenPlugin/churarenAlchemyItems/trapPlugin/domain/trap.ts

```diff
@@ -0,0 +1,35 @@
+import { Position, Direction, Entity } from 'churaverse-engine-server'
+import { ICollidableEntity } from '@churaverse/collision-detection-plugin-server/domain/collisionDetection/collidableEntity/ICollidableEntity'
+import { IRectangle } from '@churaverse/collision-detection-plugin-server/domain/collisionDetection/collidableEntity/IRectangle'
+import { ChurarenWeaponEntity } from '@churaverse/churaren-core-plugin-server'
+
+export class Trap extends Entity implements ICollidableEntity, ChurarenWeaponEntity {
+  public isCollidable = true
+  public getRect(): IRectangle {
+    return {
+      width: this._width,
+      height: this._height,
```

Comment:
当たり判定のサイズは正方形だと思うので、プロパティに渡さずにそのまま`SIDE_LENGTH`でよさそうです

```suggestion
      width: this.SIDE_LENGTH,
      height: this.SIDE_LENGTH,
```